modified_files/net/net.gypi

```diff
@@ -817,6 +817,8 @@
       'http2/hpack/decoder/hpack_block_decoder.h',
       'http2/hpack/decoder/hpack_decoder_string_buffer.cc',
       'http2/hpack/decoder/hpack_decoder_string_buffer.h',
+#      'http2/hpack/decoder/hpack_decoder_tables.cc',
```

Comment:
The files will also get added in by the script the next time it's run. The script copies over the net directory entirely, so they'll be in the repo. Since they're commented out in the build files, they won't increase the size of the binary.